pom.xml

```diff
@@ -58,19 +58,25 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <id>gitlab-maven</id>
+            <url>https://gitlab.com/api/v4/projects/37429227/packages/maven</url>
```

Comment:
I wished they didn't have this number but a company namespace or something.

Reply:
There are actually two more approachs to group the package registries. One is to add a group where all related repositories go into. And another is to add our general instance, if we would host our own at some point: [https://docs.gitlab.com/ee/user/packages/maven_repository/#use-the-gitlab-endpoint-for-maven-packages](https://docs.gitlab.com/ee/user/packages/maven_repository/#use-the-gitlab-endpoint-for-maven-packages) We could still discuss, if we prefer one of these other approaches. I thought, this one is the most straight forward. Especially we can find all dependencies in one place. But maybe we could wrap a group around this repository, so the URL looks nicer. 
Also there is the idea to mirror all dependencies to central maven repo. Then we could use that URL instead..